augur/export_v2.py

```diff
@@ -355,12 +355,16 @@ def _add_title_and_type(coloring):
             return False # a warning message will have been printed before `InvalidOption` is raised
         return coloring
 
-    def _create_coloring(key):
+    def _add_coloring(colorings, key):
         # handle deprecations
         if key == "authors":
             deprecated("[colorings] The 'authors' key is now called 'author'")
             key = "author"
-        return {"key": key}
+        # check if the key has already been added by another part of the color-creating logic
+        if key in {x['key'] for x in colorings}:
+            return False
+        colorings.append({"key": key})
+        return True
```

Comment:
Can we drop the boolean return values in this section? We don't use them in the calling code, so this code could be simplified to:

```python
if key not in {x['key'] for x in colorings}:
    colorings.append({"key": key})
```

Reply:
Yup! [diff of changes](https://github.com/nextstrain/augur/compare/a8f145167cd3afa54708e634377f326fd3b48cd7..9e9d4a40c53e84c79d9bf0e6889a599f36605192)